rules/using-passphrases/rule.md

```diff
@@ -26,19 +26,25 @@ To choose a good passphrase, use a combination of words that are unique and memo
 You might think a favorite sentence from a book might be better, given that it’s longer. While this is true in the context of time taken to brute force a password with procedural character combinations, as attackers adapt to their techniques to longer passwords, it’s important to remember that *a combination of words known to anyone in the world other than you is bad to use as a password.*
```

Comment:
@hashemelhossainissw  
I think the sentence in \* should be bold not italic, agree?

Reply:
sure, changed.